lib/internal/modules/helpers.js

```diff
@@ -354,7 +380,7 @@ module.exports = {
   loadBuiltinModule,
   makeRequireFunction,
   normalizeReferrerURL,
-  tsParse,
+  stripTypes,
```

Comment:
A little bit verbosity doesn't harm anyone

```suggestion
  stripTypescriptTypes,
```